src/frontend/src/handler/explain.rs

```diff
@@ -248,6 +252,11 @@ pub async fn handle_explain(
     if analyze {
         bail_not_implemented!(issue = 4856, "explain analyze");
     }
+    if (options.trace || options.explain_type == ExplainType::DistSql)
+        && options.explain_format == ExplainFormat::Json
+    {
+        bail_not_implemented!("explain (trace, distsql, format json) is not supported");
```

Comment:
nit: should it be `not_implemented` or `not_supported`?

besides, the error message looks like `trace && distsql && format json`

Reply:
Fixed, used the `not supported` error variant.